SwiftFilePath/Path.swift

```diff
@@ -6,21 +6,21 @@
 //  Copyright (c) 2015年 Norihiro Sakamoto. All rights reserved.
 //
 
-public class Path {
+open class Path {
     
     // MARK: - Class methods
     
-    public class func isDir(path:NSString) -> Bool {
+    open class func isDir(_ path:String) -> Bool {
         var isDirectory: ObjCBool = false
-        _ = NSFileManager.defaultManager().fileExistsAtPath(path as String, isDirectory:&isDirectory)
+        _ = FileManager.default.fileExists(atPath: path, isDirectory:&isDirectory)
```

Comment:
I guess I can use this other one: https://developer.apple.com/reference/foundation/filemanager/1415645-fileexists